tools/antithesis-go-instrumentor/cmd/cmd_files.go

```diff
@@ -312,3 +345,37 @@ func (cfx *CommandFiles) CreateSymbolTableWriter(filesHash string) (symWriter *i
 func (cfx *CommandFiles) GetNotifierDirectory() string {
 	return cfx.notifierDirectory
 }
+
+func (cfx *CommandFiles) ShowDependentModules() {
```

Comment:
these functions are not recursive.  Can dependent modules have recursive dependencies?

Reply:
dependent modules can not have recursive dependencies